.github/workflows/test-environment.yml

```diff
@@ -98,11 +98,11 @@ on:
         type: string
         description: "**Optional** By default, the environment will be created in our Cloud Security Organization. If you want to use your own cloud account, enter your Elastic Cloud API key."
         required: false
-      cdr-infra:
-        description: "Flag to indicate that the CDR infrastructure is being created"
-        type: boolean
+      infra-type:
+        description: "Type of infrastructure to create"
+        type: string
         required: false
-        default: false
+        default: "cis"
```

Comment:
maybe a boolean for CDR and CIS is better? otherwise the string needs to be known to whoever runs this workflow

Reply:
This will be resolved by refactoring the test-environment workflow in [this task](https://github.com/elastic/security-team/issues/10815).